lib/helpers/validateSignupData.js

```diff
@@ -0,0 +1,32 @@
+import Schema from 'validate'
+
+const validateSignupData = data => {
+  const userData = new Schema({
```

Comment:
you can take the `Schema` definition outside the function. this will fix the codeClimate issue.